package.json

```diff
@@ -21,23 +21,27 @@
     "npm": "8.11.0"
   },
   "dependencies": {
+    "bcrypt": "^5.1.0",
```

Comment:
Please use fixed version of dependencies to avoid conflict version of relative deps